cli/src/main/java/com/box/l10n/mojito/cli/command/DateTimeConverter.java

```diff
@@ -1,11 +1,20 @@
 package com.box.l10n.mojito.cli.command;
 
 import com.beust.jcommander.IStringConverter;
-import org.joda.time.DateTime;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeParseException;
+
+public class DateTimeConverter implements IStringConverter<ZonedDateTime> {
 
-public class DateTimeConverter implements IStringConverter<DateTime> {
   @Override
-  public DateTime convert(String dateAsText) {
-    return DateTime.parse(dateAsText);
+  public ZonedDateTime convert(String dateAsText) {
+    try {
+      return ZonedDateTime.parse(dateAsText);
```

Comment:
what are the difference between the 2 `parse()` implementation? Is this documented somewhere? Is the default `DateTimeFormatter` different? If different, is possible to specify one that is fully backward compatible with the Joda's one?

Reply:
The main difference is that Joda. DateTime works with or without a time zone specified, while ZonedDateTime timezone is mandatory. There is no class in java which is fully backward compatible. What I am doing in this converter to keep the backward compatibility is: 
Trying to parse the string as a ZonedDateTime, and if not possible, parse it as a LocalDateTime (which does not requires time zone) and setting UTC as the timezone. By doing this I am keeping the same behavior we had previously for joda.DateTime